doc/developers-guide/mod_muc_light_developers_guide.md

```diff
@@ -27,16 +27,29 @@ All source files can be found in `apps/ejabberd/src`.
   Not recommended for production systems (less efficient than modern codec and requires more round-trips).
 
 * `mod_muc_light_codec_modern.erl`
-  An implementation of modern MUC protocol, described in the XEP. 
+  An implementation of a modern MUC Light protocol, described in the XEP.
   Supports all MUC Light features.
 
+* `mod_muc_light_commands.erl`
+  MUC Light-related commands.
+  They are registered in the `mongoose_commands` module, so they are available via the REST API.
+
 * `mod_muc_light_db.erl`
   A behaviour implemented by database backends for the MUC Light extension.
 
 * `mod_muc_light_db_mnesia.erl`
-  Mnesia backend for this extension. 
+  A Mnesia backend for this extension.
```

Comment:
I would leave the article out here, but I'm not 100% sure, we better consult a native speaker.

Reply:
native said to keep the article:) he also said to put an "An" there but i think he's pronouncing it "amnesia;))"